### DISCUSSION TITLE
Split outputs?

### DISCUSSION THREAD

**Discussion by NlGHTLORD** [opentrack/opentrack#1567] [Q&A]
I tried created 1 udp output to two upd inputs but a third instance of open track does not seem to run.  I need a freetrack to move my head around in game and another output to VJoy so I can create more comples macros depending on where I am looking (using Joystick Gremlin)

Also is there a way in the VJoy output to control which virtual device the information is sent to.  It seems to automatically control Vjoy 1 regardless of how many virtual devices there are.

Thank you!

**Comment by NlGHTLORD** [comment]
So I did manage to make two out puts by having two instances of OpenTrack running. One with PointTracker sending data to Freetrack and the second with Aruco sending data to Vjoy.  Works but there is a lot of stuff on my head.

**Reply by NlGHTLORD** [reply]
No, this really doesn't work so well.  I need dual outputs. Any takers, I'll give you $20 bucks.

**Reply by sthalik** [reply]
This is actually easy to implement (mostly a copy of what tracker-fusion does) but contractor rates are different by an order of magnitude. But seriously though, I'm busy with a gamedev project and the next new opentrack feature is meant to be OSC output support.

**Reply by NlGHTLORD** [reply]
So I did find the fusion thread and the files.  I have an ok grasp on AHK but not c++ anywhere near the level you guys are. Where do I place these files to compile?

**Reply by NlGHTLORD** [reply]
NM, Fusion is for two separate inputs, not outputs.  Looking to be able to run Joystick Emulation Output and FreeTrack 2.0 Enhanced together if anyone has any ideas.

**Reply by sthalik** [reply]
What I said is `tracker-fusion` can be used as a reference as to how a split protocol (aka output) has to look like.

**Comment by NlGHTLORD** [comment]
Oh, got it! Use it as a base to write something or reverse engineer it.
I'll dabble and maybe learn something. Thanks.
